pom.xml

```diff
@@ -124,6 +124,13 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <configuration>
+          <excludeFilterFile>src/findbugs/excludesFilter.xml</excludeFilterFile>
```

Comment:
this shouldn't be needed :)

You should rename the folder I believe
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/plugin/src/spotbugs/excludesFilter.xml

Reply:
I didn't know that before. It's cool.

Reply:
I suggest dropping `Add spotbugs exclude filters` :)